openlibrary/templates/site/head.html

```diff
@@ -12,8 +12,8 @@
     <meta name="author" content="OpenLibrary.org" />
     <meta name="creator" content="OpenLibrary.org" />
     <meta name="copyright" content="Original content copyright; 2007-2015" />
-    <meta name="distribution" content="Global" />
-
+    <meta name="distribution" content="Global, India" />
```

Comment:
Also, I love the idea but global local and iu are the only valid fields for distribution!